Formula/spider.rb

```diff
@@ -0,0 +1,24 @@
+class Spider < Formula
+  desc "small dart library to generate Assets dart code from assets folder."
+  homepage "https://github.com/BirjuVachhani/spider"
+  url "https://github.com/BirjuVachhani/spider/archive/1.0.1.tar.gz"
+  sha256 "cb3d3af95888551ebacb76f5d7a06d196e43417d25980c84f235d5ad4f2baded"
+  license "Apache-2.0"
+  
+  bottle :unneeded
+  
+  depends_on "dart-lang/dart/dart" => :build
```

Comment:
Homebrew-core formulae can only depend on homebrew-core formula. You'll have to migrate dart to homebrew-core first.

Reply:
Well, this seems to be blocker. I have submitted an issue for this on dart-lang/dart formulae but it seems that they won't do it. Closing this PR for now as I am not expecting them to make dart SDK available in homebrew core.

Thanks @SMillerDev

Reply:
It's an open source tool right? You can add it to homebrew as well.

Reply:
It is. But they are maintaining their own tap so I need to know the reason behind it. If they won't then I will if possible.